content/tutorials/publish-subscribe.textile

```diff
@@ -245,6 +335,37 @@ h2(#live-demo). Live demo
 
 h2. Download tutorial source code
 
+blang[java].
+  The complete source code for each step of "this tutorial is available on Github":https://github.com/piotrekinventica/tutorials/commits/publish-subscribe-jre.
+
+  We recommend that you clone the repo locally:
+
+  bc[sh]. git clone https://github.com/ably/tutorials.git
+
+  Checkout the tutorial branch:
+
+  bc[sh]. git checkout publish-subscribe-jre
```

Comment:
I would argue we should use the naming `java-jre` in fact instead of just `jre` as anyone browsing the branches would not necessarily know what JRE stands for...